Examples/ArcGISToolkitExamples/JobManagerExample.swift

```diff
@@ -139,6 +139,9 @@ class JobManagerExample: TableViewController {
         
         // job cell registration
         tableView.register(JobTableViewCell.self, forCellReuseIdentifier: "JobCell")
+        
+        // resume any paused jobs
+        JobManager.shared.resumeAllPausedJobs(statusHandler: self.jobStatusHandler, completion: self.jobCompletionHandler)
```

Comment:
I realize this isn't your code, but there's a subtle bug here. By passing methods as the closures, a strong reference to `self` is being retained, preventing the view controller from being deinitialized. I'm guessing that's why I got a prompt for credentials back on the main view:

<img width="584" alt="image" src="https://user-images.githubusercontent.com/2257493/71750964-e3983d80-2e2e-11ea-990c-45c2d3dd71e1.png">

Either way, the solution is to pass closures with `weak` references to `self` that call the appropriate methods rather than pass the methods themselves. The same is true wherever a job is started (lines `275` and `301`. If you don't want to fix it yourself, I can submit a PR after yours is merged.

Reply:
good point - I can fix this - thank you

Reply:
Thank you for fixing that!